src/librawspeed/decompressors/CrxDecompressor.cpp

```diff
@@ -450,14 +450,15 @@ inline void crxDecodeSymbolL1Rounded(CrxBandParam* param, int32_t doSym = 1,
   param->lineBuf1[1] = param->roundedBitsMask * 2 * code + (code >> 31) + sym;
 
   if (doCode) {
-    if (param->lineBuf0[2] > param->lineBuf0[1])
+    if (param->lineBuf0[2] > param->lineBuf0[1]) {
       code = (param->lineBuf0[2] - param->lineBuf0[1] + param->roundedBitsMask -
               1) >>
              param->roundedBits;
-    else
+    } else {}
```

Comment:
@johnny-bit Is this correct?

Reply:
no, this is very incorect. How tf it got here? (the file is outdated, new commit fixed that btw)